app/kumactl/cmd/uninstall/uninstall_transparent_proxy_test.go

```diff
@@ -37,20 +34,11 @@ var _ = Describe("kumactl install tracing", func() {
 			// when
 			err := rootCmd.Execute()
 			// then
-			Expect(err).ToNot(HaveOccurred())
+			Expect(err).To(HaveOccurred())
 			// and
-			Expect(stderr.String()).To(BeEmpty())
+			Expect(stderr.String()).To(Equal("Error: transparent proxy cleanup failed: cleanup is not supported\n"))
```

Comment:
we don't support uninstall in v2 TP?

Reply:
Yeah we don't IMO it's silly to run this directly on your host network...

Reply:
👍 - maybe might make sense to put this in docs

Reply:
Yes I'll add it